api/v1alpha1/httproutefilter_types.go

```diff
@@ -72,6 +72,13 @@ type HTTPDirectResponseFilter struct {
 	// If unset, defaults to 200.
 	// +optional
 	StatusCode *int `json:"statusCode,omitempty"`
+
+	// ResponseHeadersToAdd defines headers to add to the response.
+	// This allows the response policy to append, add or override headers
+	// of the original response before it is sent to a downstream client.
+	//
+	// +optional
+	ResponseHeadersToAdd []ResponseHeaderToAdd `json:"responseHeadersToAdd,omitempty"`
```

Comment:
Since this field is inside the `HTTPDirectResponseFilter` struct, prefer just naming it `Headers`.